tool/plugin/lib/plugin.dart

```diff
@@ -271,6 +283,15 @@ void separator(String name) {
   log('${ansi.red}${ansi.bold}$name${ansi.none}', indent: false);
 }
 
+int get devBuildNumber {
```

Comment:
Do we have the ability to publish a rev to a weekly dev build? What if we find a P0 in the dev channel and what to patch it in order to help validate a fix? a dev channel `.1`

Reply:
We don't have that currently. We can add it if needed. We'd probably have to add a --patch option or similar, and it should work for stable, master, and dev builds.